### PR TITLE
Fix Brand Colors

### PR DESCRIPTION
**Pull request by danrabbit**
There's some serious fuckery afoot that breaks brand colors pretty bad

This should fix app set brand colors for checkboxes, selections, progressbars, etc

### CODE REVIEW

File: src/variants/banana-dark.scss [hunk 1]
```diff
@@ -26,7 +26,7 @@
 @define-color accent_color_700 @BANANA_700;
 @define-color accent_color_900 @BANANA_900;
 
-@define-color accent_color @BANANA_500;
+@define-color accent_color mix (@BANANA_500, @BANANA_700, 0.25);
```
Comment (cassidyjames) [inline src/variants/banana-dark.scss:29]: nitpick, but typically in (S)CSS, functions don't have whitespace before parens

```suggestion
@define-color accent_color mix(@BANANA_500, @BANANA_700, 0.25);
```

File: src/variants/blueberry-dark.scss [hunk 2]
```diff
@@ -26,7 +26,7 @@
 @define-color accent_color_700 @BLUEBERRY_700;
 @define-color accent_color_900 @BLUEBERRY_900;
 
-@define-color accent_color #{'mix(@accent_color_300, @accent_color_500, 0.5)'};
+@define-color accent_color mix (@BLUEBERRY_300, @BLUEBERRY_500, 0.5);
```
Comment (cassidyjames) [inline src/variants/blueberry-dark.scss:29]:
```suggestion
@define-color accent_color mix(@BLUEBERRY_300, @BLUEBERRY_500, 0.5);
```

File: src/variants/blueberry.scss [hunk 3]
```diff
@@ -26,7 +26,7 @@
 @define-color accent_color_700 @BLUEBERRY_700;
 @define-color accent_color_900 @BLUEBERRY_900;
 
-@define-color accent_color @BLUEBERRY_700;
+@define-color accent_color mix (@BLUEBERRY_300, @BLUEBERRY_500, 0.25);
```
Comment (cassidyjames) [inline src/variants/blueberry.scss:29]:
```suggestion
@define-color accent_color mix(@BLUEBERRY_300, @BLUEBERRY_500, 0.25);
```

File: src/variants/bubblegum-dark.scss [hunk 4]
```diff
@@ -26,7 +26,7 @@
 @define-color accent_color_700 @BUBBLEGUM_700;
 @define-color accent_color_900 @BUBBLEGUM_900;
 
-@define-color accent_color #{'mix(@accent_color_300, @accent_color_500, 0.5)'};
+@define-color accent_color mix (@BUBBLEGUM_300, @BUBBLEGUM_500, 0.5);
```
Comment (cassidyjames) [inline src/variants/bubblegum-dark.scss:29]:
```suggestion
@define-color accent_color mix(@BUBBLEGUM_300, @BUBBLEGUM_500, 0.5);
```

File: src/variants/bubblegum.scss [hunk 5]
```diff
@@ -26,7 +26,7 @@
 @define-color accent_color_700 @BUBBLEGUM_700;
 @define-color accent_color_900 @BUBBLEGUM_900;
 
-@define-color accent_color @BUBBLEGUM_700;
+@define-color accent_color mix (@BUBBLEGUM_300, @BUBBLEGUM_500, 0.25);
```
Comment (cassidyjames) [inline src/variants/bubblegum.scss:29]:
```suggestion
@define-color accent_color mix(@BUBBLEGUM_300, @BUBBLEGUM_500, 0.25);
```

File: src/variants/cocoa-dark.scss [hunk 6]
```diff
@@ -26,7 +26,7 @@
 @define-color accent_color_700 @COCOA_700;
 @define-color accent_color_900 @COCOA_900;
 
-@define-color accent_color #{'mix(@accent_color_100, @accent_color_300, 0.5)'};
+@define-color accent_color mix (@COCOA_300, @COCOA_500, 0.25);
```
Comment (cassidyjames) [inline src/variants/cocoa-dark.scss:29]:
```suggestion
@define-color accent_color mix(@COCOA_300, @COCOA_500, 0.25);
```

File: src/variants/grape.scss [hunk 7]
```diff
@@ -26,7 +26,7 @@
 @define-color accent_color_700 @GRAPE_700;
 @define-color accent_color_900 @GRAPE_900;
 
-@define-color accent_color @GRAPE_700;
+@define-color accent_color mix (@GRAPE_300, @GRAPE_500, 0.5);
```
Comment (cassidyjames) [inline src/variants/grape.scss:29]:
```suggestion
@define-color accent_color mix(@GRAPE_300, @GRAPE_500, 0.5);
```

File: src/variants/lime-dark.scss [hunk 8]
```diff
@@ -26,7 +26,7 @@
 @define-color accent_color_700 @LIME_700;
 @define-color accent_color_900 @LIME_900;
 
-@define-color accent_color #{'mix(@accent_color_300, @accent_color_500, 0.5)'};
+@define-color accent_color mix (@LIME_300, @LIME_500, 0.5);
```
Comment (cassidyjames) [inline src/variants/lime-dark.scss:29]:
```suggestion
@define-color accent_color mix(@LIME_300, @LIME_500, 0.5);
```

File: src/variants/lime.scss [hunk 9]
```diff
@@ -26,7 +26,7 @@
 @define-color accent_color_700 @LIME_700;
 @define-color accent_color_900 @LIME_900;
 
-@define-color accent_color #{'mix(@accent_color_700, @accent_color_900, 0.5)'};
+@define-color accent_color mix (@LIME_300, @LIME_500, 0.25);
```
Comment (cassidyjames) [inline src/variants/lime.scss:29]:
```suggestion
@define-color accent_color mix(@LIME_300, @LIME_500, 0.25);
```

File: src/variants/mint-dark.scss [hunk 10]
```diff
@@ -26,7 +26,7 @@
 @define-color accent_color_700 @MINT_700;
 @define-color accent_color_900 @MINT_900;
 
-@define-color accent_color #{'mix(@accent_color_300, @accent_color_500, 0.5)'};
+@define-color accent_color mix (@MINT_500, @MINT_700, 0.25);
```
Comment (cassidyjames) [inline src/variants/mint-dark.scss:29]:
```suggestion
@define-color accent_color mix(@MINT_500, @MINT_700, 0.25);
```

File: src/variants/mint.scss [hunk 11]
```diff
@@ -26,7 +26,7 @@
 @define-color accent_color_700 @MINT_700;
 @define-color accent_color_900 @MINT_900;
 
-@define-color accent_color @MINT_700;
+@define-color accent_color mix (@MINT_300, @MINT_500, 0.75);
```
Comment (cassidyjames) [inline src/variants/mint.scss:29]:
```suggestion
@define-color accent_color mix(@MINT_300, @MINT_500, 0.75);
```

File: src/variants/orange.scss [hunk 12]
```diff
@@ -26,7 +26,7 @@
 @define-color accent_color_700 @ORANGE_700;
 @define-color accent_color_900 @ORANGE_900;
 
-@define-color accent_color @ORANGE_700;
+@define-color accent_color mix (@ORANGE_300, @ORANGE_500, 0.5);
```
Comment (cassidyjames) [inline src/variants/orange.scss:29]:
```suggestion
@define-color accent_color mix(@ORANGE_300, @ORANGE_500, 0.5);
```

File: src/variants/slate-dark.scss [hunk 13]
```diff
@@ -26,7 +26,7 @@
 @define-color accent_color_700 @SLATE_700;
 @define-color accent_color_900 @SLATE_900;
 
-@define-color accent_color #{'mix(@accent_color_100, @accent_color_300, 0.5)'};;
+@define-color accent_color mix (@SLATE_100, @SLATE_300, 0.5);
```
Comment (cassidyjames) [inline src/variants/slate-dark.scss:29]:
```suggestion
@define-color accent_color mix(@SLATE_100, @SLATE_300, 0.5);
```

File: src/variants/strawberry.scss [hunk 14]
```diff
@@ -26,7 +26,7 @@
 @define-color accent_color_700 @STRAWBERRY_700;
 @define-color accent_color_900 @STRAWBERRY_900;
 
-@define-color accent_color @STRAWBERRY_700;
+@define-color accent_color mix (@STRAWBERRY_300, @STRAWBERRY_500, 0.25);
```
Comment (cassidyjames) [inline src/variants/strawberry.scss:29]:
```suggestion
@define-color accent_color mix(@STRAWBERRY_300, @STRAWBERRY_500, 0.25);
```

File: src/widgets/_checkbuttons.scss [hunk 15]
```diff
@@ -12,10 +12,14 @@ radio {
     -gtk-icon-source: none;
 
     &:checked {
-        background-color: #{'@accent_color_300'};
+        background-color: #{'@accent_color'};
         color: white;
         -gtk-icon-source: -gtk-icontheme("check-active-symbolic");
-        -gtk-icon-shadow: 0 1px 1px #{'@accent_color_500'};
+        -gtk-icon-shadow: 0 1px 1px #{'shade (@accent_color, 0.7)'};
```
Comment (cassidyjames) [inline src/widgets/_checkbuttons.scss:18]:
```suggestion
        -gtk-icon-shadow: 0 1px 1px #{'shade(@accent_color, 0.7)'};
```

File: src/_exported.scss [hunk 16]
```diff
@@ -24,15 +24,15 @@
...
+@define-color selected_bg_color #{'alpha(@accent_color, 0.25)'};
+@define-color selected_fg_color #{'shade(@accent_color, 0.5)'};
 
 @if $color-scheme == "dark" {
     @define-color menu_separator #{rgba(black, 0.25)};
     @define-color menu_separator_shadow #{rgba(white, 0.05)};
 
-    @define-color selected_bg_color #{'alpha(@accent_color_700, 0.3)'};
-    @define-color selected_fg_color #{'@accent_color_100'};
+    @define-color selected_bg_color #{'alpha (shade(@accent_color, 0.85), 0.5)'};
```
Comment (cassidyjames) [inline src/_exported.scss:34]:
```suggestion
    @define-color selected_bg_color #{'alpha(shade(@accent_color, 0.85), 0.5)'};
```

File: src/widgets/_checkbuttons.scss (hunk 15 above)
Comment (danrabbit) [reply src/widgets/_checkbuttons.scss:18]: this is a Gtk func, not sure if spacing is important here

**Comment by cassidyjames** [comment]
@danrabbit sorry, I should have done those all as one review; I didn't realize it was notifying on every change! Reviewing the actual behavior now…

**Review by cassidyjames — APPROVED** [review]

**Comment by cassidyjames** [comment]
Yay accent colors!

**Pull request merged**